docs/spec/sanitations.md

```diff
@@ -5,23 +5,34 @@ _Edition_: Swan Lake
+This document outlines the modifications and enhancements made to the OpenAPI specification for `salesforce.marketingcloud`.
+Since there is no official OpenAPI specification, the spec was generated by converting the [Salesforce Marketing Cloud Postman Collection](https://www.postman.com/salesforce-developers/salesforce-developers/folder/hs55w6u/rest). While the Postman collection provided a solid foundation, it lacked many parameters and response structures. These gaps were filled by manually adding the required details and making further adjustments to improve usability and address language-specific limitations.
 
 **Note:** Only a subset of APIs was implemented, as adding all parameter and response types required significant manual effort.
 
-Sanitation steps applied:
-1. Added `OAuth2` credential security schema.
+### Sanitation Steps Applied
+
+1. Added the `OAuth2` credential security schema.
 2. Retained only the endpoints selected for implementation.
 3. Removed authentication endpoints.
 4. Removed discovery endpoints.
-5. Added parameters and response structures, keeping only the minimum required fields with `additionalParameters` set to true.
+5. Added parameters and response structures, keeping only the minimum required fields with `additionalProperties` set to true.
 6. Ran `bal openapi align` and `bal openapi flatten` commands.
-7. Add `operationId` for the paths.
- 
-## OpenAPI cli command
+7. Added `operationId` for all paths.
+
+## Adding New APIs
+
+To add a new API to the specification and client:
+
+1. Locate the relevant API documentation page in the [SFMC documentation](https://developer.salesforce.com/docs/marketing/marketing-cloud/guide/apis-overview.html).
+2. Add the endpoint and related schemas to the `docs/spec/openapi.json` file.
+   > **Tip:** Review both the Guides and Reference pages, as Guides often contain additional details not present in the Reference examples.
+3. Generate the client using the command in the [OpenAPI CLI Command](#openapi-cli-command) section.
+4. Apply the sanitations described in [Sanitation for Generated Client](#sanitation-for-generated-client).
+
+## OpenAPI CLI Command
 
-The following command was used to generate the Ballerina client from the OpenAPI specification. The command should be executed from the repository root directory.
+Use the following command to generate the Ballerina client from the OpenAPI specification. Run this from the repository root:
 
 ```bash
 bal openapi -i docs/spec/openapi.json -o ballerina --mode client --client-methods remote --license docs/license.txt
```

Comment:
Shall we provide the reasoning behind using `remote` methods, since our default is `resource` methods?